addons/website_mail_channel/controllers/main.py

```diff
@@ -1,12 +1,13 @@
 # -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-import datetime
+from datetime import datetime
 from dateutil import relativedelta
 
-from openerp import tools, SUPERUSER_ID
-from openerp.addons.web import http
-from openerp.addons.website.models.website import slug
-from openerp.addons.web.http import request
+from odoo import http
+from odoo.fields import Datetime
```

Comment:
I would import fields and use fields.Datetime in the code, to avoid confusing with datetime itself.